corehq/apps/users/templates/users/email/notify_scheduled_user_migration.html

```diff
@@ -4,31 +4,30 @@
 
 {% blocktrans %}
   <p>
-    We have scheduled a migration to be run on {{ scheduled_date }} that will impact your CommCare user. Your
-    organization has requested that your user, {{ old_username }}, be migrated to a new user with the username
-    {{ new_username }}.
-
+    We have scheduled a migration to be run on {{ scheduled_date }} that will impact your CommCare user. MSI has requested that your user, {{ old_username }}, be migrated to a new user with the username {{ new_username }}.
```

Comment:
Are we planning to remove these templates after using them for MSI? It doesn't seem right to encode a specific customer name here.

The same applies to other templates as well.